test/external/channel-benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+include(NeuronTestHelper)
+set(prefix "benchmark/channels") # everything is under this prefix in the repository
+set(channel_benchmark_sim_time 10)
+set(channel_benchmark_mpi_ranks 1)
```

Comment:
using single rank, what is simulation time for the 10msec simulation? and how many spikes are there? (just be sure there is activity in the network)

Reply:
Hippocampus:
```
 Simulation Statistics
 Number of cells: 6
 Number of compartments: 4148
 Number of presyns: 18006
 Number of input presyns: 0
 Number of synapses: 18001
 Number of point processes: 36007
 Number of transfer sources: 0
 Number of transfer targets: 0
 Number of spikes: 6
 Number of spikes with non negative gid-s: 6
```
SSCx:
```
 Simulation Statistics
 Number of cells: 16
 Number of compartments: 6658
 Number of presyns: 48016
 Number of input presyns: 0
 Number of synapses: 48001
 Number of point processes: 96017
 Number of transfer sources: 0
 Number of transfer targets: 0
 Number of spikes: 33
 Number of spikes with non negative gid-s: 33
```
in all cases the simulation takes 3-4s to run.